utils/metrics/metrics_util.go

```diff
@@ -104,3 +104,15 @@ func MeasureDeliverTxDuration(start time.Time) {
 		start.UTC(),
 	)
 }
+
+// sei_oracle_vote_penalty_count
+func SetOracleVotePenaltyCount(count uint64, valAddr string, penaltyType string) {
+	metrics.SetGaugeWithLabels(
```

Comment:
btw, not sure exactly how this behaves vs `incr`, but are we able to get the combined vote count if we query with a `validator` but without a `type`? because the slashing is based on the combined value so would be helpful to have, or if we can't then maybe just add a `combined` label that just adds the two vals

Reply:
as discussed offline, we can sum by the validator